pt/tutorials-and-examples/blog/blog.rst

```diff
@@ -1,54 +1,58 @@
-Blog Tutorial
-#############
-
-This tutorial will walk you through the creation of a simple blog application.
-We'll be installing CakePHP, creating a database, and creating enough
-application logic to list, add, edit, and delete blog posts.
-
-Here's what you'll need:
-
-#. A running web server. We're going to assume you're using Apache,
-   though the instructions for using other servers should be very
-   similar. We might have to play a little with the server
-   configuration, but most folks can get CakePHP up and running without
-   any configuration at all. Make sure you have PHP 5.4.16 or greater, and
-   that the ``mbstring`` and ``intl`` extensions are enabled in PHP.
-#. A database server. We're going to be using MySQL server in this
-   tutorial. You'll need to know enough about SQL in order to create a
-   database: CakePHP will be taking the reins from there. Since we're using MySQL,
-   also make sure that you have ``pdo_mysql`` enabled in PHP.
-#. Basic PHP knowledge.
-
-Let's get started!
-
-Getting CakePHP
-===============
-
-The easiest way to install CakePHP is to use Composer.  Composer is a simple way
-of installing CakePHP from your terminal or command line prompt.  First, you'll
-need to download and install Composer if you haven't done so already. If you
-have cURL installed, it's as easy as running the following::
+Construindo um blog
+###################
+
+Este tutorial irá orientá-lo na criação de um simples aplicativo de blog. 
+Estaremos instalando CakePHP, criando um banco de dados e criando a lógica de aplicação suficiente para listar, 
+adicionar, editar e apagar posts.
+
+Aqui está o que você vai precisar:
+
+#. Um servidor web rodando. Vamos supor que você está usando o Apache, 
+   embora as instruções para usar outros servidores sejam muito semelhantes. 
+   Poderíamos ter que brincar um pouco com a configuração do servidor, 
+   mas a maioria das pessoas pode ter o CakePHP instalado e funcionando sem 
+   qualquer configuração em tudo. 
+   Certifique-se de que você tem o PHP 5.4.16 ou superior, e que os ``mbstring`` e ``intl`` 
+   extensões são ativadas no PHP.
+#. Um servidor de banco de dados. Nós vamos estar usando servidor MySQL neste tutorial. 
```

Comment:
_Nós iremos utilizar um servidor MySQL neste tutorial._
You are free to adapt the translation pro readability.